config/locales/en.yml

```diff
@@ -583,6 +593,18 @@ en:
       end_date: End date
       month: Weeks of the Month
       period: Weekly interval
+  podcast_player:
+    form:
+      help:
+        <<: *player_help
```

Comment:
basically using the same `help` and `title` text from the Episode Embed Player text above. then added specific text needs below. Probably need someone to look at verbiage?